### ISSUE TITLE
[question] Can use `xgboost` or `lightgbm`

### ISSUE THREAD

**Issue by Liripo** [aertslab/pySCENIC#550]
The `grn` step uses sklearn `gradient boosting regularization`. Can `xgboost` or `lightgbm` be used instead? What are the parameters like?

**Comment by liangyu02** [comment]
very good
